### PR TITLE
Adding seeds to database

### LINKED ISSUE #4: Image Seeds
#### As a developer I want at least 20 images to be created when I setup the database.

__Story__:

Working on this application has been a lot of fun, right? You are fast
becoming a git expert, and an expert at working with a deployment
pipeline. Speaking of development and deployment pipelines, is it not pretty
annoying that everytime a new deployment is created it starts out with an empty
database? It seems like everytime a review application is created we need to
manually add a handful of images in order to manually test the
application. Would it not be great if each deployment was created with a
handful of existing images?

__Acceptance criteria__:
- [x] After running `rake db:setup` at least 20 unique images are shown in the
  homepage.
- [x] When a new app is deployed on heroku (e.g., every review app), that app
  automatically has at least 20 unique images on its homepage.

__Dependencies__:
- Image Index

### PR DESCRIPTION
**Pull request by dhanson11**
Closes #4

### CODE REVIEW

File: app/views/application/home.html.erb [hunk 1]
```diff
@@ -4,11 +4,12 @@
 <%= link_to 'Click here to add a new image', new_image_path %>
 
 <h1>Listing all images</>
-
+<p>
```
Comment (YJ83Lee) [inline app/views/application/home.html.erb]: Maybe we can move this code changes into a separate commit since it is not relevant to adding seeds, which I suppose is the main purpose of this commit?

File: db/seeds.rb [hunk 2]
```diff
@@ -5,3 +5,25 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+Image.create(url: 'https://d1yjjnpx0p53s8.cloudfront.net/styles/logo-thumbnail/s3/122012/49ers.jpg?itok=RPNisAtV')
```
Comment (YJ83Lee) [inline db/seeds.rb]: Just FYI.
`create()` can accept an array of hash as parameters to create multiple records.

https://api.rubyonrails.org/classes/ActiveRecord/Persistence/ClassMethods.html#method-i-create
```ruby
# Create an Array of new objects
User.create([{ first_name: 'Jamie' }, { first_name: 'Jeremy' }])
```

File: app/views/application/home.html.erb (hunk 1 above)
Comment (dhanson11) [reply app/views/application/home.html.erb]: will do

**Comment by dhanson11** [comment]
I believe this is ready!
